### PR TITLE
shadowfinder: add package

### PR DESCRIPTION
**Pull request by D3vil0p3r**
Close https://github.com/BlackArch/blackarch/issues/4194

### CODE REVIEW

File: packages/shadowfinder/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+pkgver=0.5.0.r0.g578d5ac
+pkgrel=1
+pkgdesc='Find possible locations of shadows around the world.'
+arch=('any')
+url='https://github.com/bellingcat/ShadowFinder'
+license=('MIT')
+depends=('python' 'python-basemap' 'python-matplotlib' 'python-fire' 'python-numpy' 'python-pandas' 'python-pytz' 'python-suncalc' 'python-timezonefinder')
+makedepends=('git' 'python-build' 'python-pip')
+options=(!emptydirs)
+source=("git+https://github.com/bellingcat/$_pkgname.git")
```
Comment (noraj) [inline packages/shadowfinder/PKGBUILD:15]:
```suggestion
source=("git+https://github.com/bellingcat/$pkgname.git")
```

Github namespaces are case insensitive, so you can use the lowercase vversion of the name for the URL and then to `cd` into the cloned folder.

File: packages/shadowfinder/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    --disable-pip-version-check \
+    --no-warn-script-location \
+    --ignore-installed \
+    --no-compile \
+    --no-deps \
+    --root="$pkgdir" \
+    --prefix=/usr \
+    --no-index \
+    --find-links="file://$startdir/dist" \
+    $_pkgname
```
Comment (noraj) [inline packages/shadowfinder/PKGBUILD:49]: If you `tar tvf` the `.zst` archive, is the binary name lowercase? If not you should `mv` it.

Comment (D3vil0p3r) [reply packages/shadowfinder/PKGBUILD]: It works also lowercase, so no need to change here.

**Comment by noptrix** [comment]
thank you!

File: packages/shadowfinder/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=shadowfinder
+pkgver=0.5.0.r0.g578d5ac
+pkgrel=1
+pkgdesc='Find possible locations of shadows around the world.'
+arch=('any')
```
[the remark this answers is not in the document]
Comment (D3vil0p3r) [reply packages/shadowfinder/PKGBUILD:8]: Fixed

File: packages/shadowfinder/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+url='https://github.com/bellingcat/ShadowFinder'
+license=('MIT')
+depends=('python' 'python-basemap' 'python-matplotlib' 'python-fire' 'python-numpy' 'python-pandas' 'python-pytz' 'python-suncalc' 'python-timezonefinder')
```
[the remark this answers is not in the document]
Comment (D3vil0p3r) [reply packages/shadowfinder/PKGBUILD]: Fixed, thanks

**Review by noptrix — APPROVED** [review]

**Pull request merged**